modules/compliance-operator-hcp-mgmt-config.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+:_content-type: PROCEDURE
+[id="co-hcp-mgmt-config_{context}"]
+= Configuring the Hosted control planes management cluster
+
+If you are hosting your own Hosted control plane or Hypershift environment and want to scan a Hosted Cluster from the management cluster, you will need to set the name and prefix namespace for the target Hosted Cluster. You can achieve this by creating a `TailoredProfile`.
+
+[IMPORTANT]
+====
+This procedure only applies to users managing their own Hosted control planes environment.
+====
+
+[NOTE]
+====
+Only `ocp4-cis` and `ocp4-pci-dss` profiles are supported in Hosted control planes management clusters.
+====
+
+.Prerequisites
+
+* The Compliance Operator is installed in the management cluster.
+
+.Procedure
+
+. Obtain the `name` and `namespace` of the hosted cluster to be scanned by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE       NAME                   VERSION   KUBECONFIG                              PROGRESS    AVAILABLE   PROGRESSING   MESSAGE
+local-cluster   79136a1bdb84b3c13217   4.13.5    79136a1bdb84b3c13217-admin-kubeconfig   Completed   True        False         The hosted control plane is available
+----
+
+. In the management cluster, create a `TailoredProfile` extending the Profile you want to scan with, and define the name and namespace of the Hosted Cluster to be scanned:
++
+.Example `management-tailoredprofile.yaml`
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: hypershift-cisk57aw88gry
+  namespace: openshift-compliance
+spec:
+  description: This profile test required rules
+  extends: ocp4-cis
```

Comment:
Generally it is look good. To make it more clear, maybe we can make the extends as variable, and clarify only `ocp4-cis` and `ocp4-pci-dss` is supported. If so, we can remove the previous NOTE.

Reply:
That's a great idea. I updated the callouts: 
![image](https://github.com/openshift/openshift-docs/assets/46728411/d9143d92-e282-4c5d-a7b9-398cfc29f2bc)

I also left the NOTE at the top to reinforce the knowledge.